### PR TITLE
docs - clean up docs on referencing question and models in SQL questions

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
- Updates image
- Drops outdated content
- Clarifies how to reference models and questions in SQL queries.

### CODE REVIEW

File: docs/questions/native-editor/referencing-saved-questions-in-queries.md [hunk 1]
```diff
@@ -8,48 +8,21 @@ redirect_from:
 
 With SQL databases, we can use a [model][model] or an existing question as the basis for a new query, or as a common table expression [CTE][CTE].
 
-For example, let's say we have a lot of data spread across a number of tables, but our users are most interested in a subset of that data. We can perform a complicated query once to return those results, and save that question as a model, which people can refer to in their queries just like they would with any other table.
+For example, let's say we have a lot of data spread across a number of tables, but our people are most interested in a subset of that data. We can perform a complicated query once to return those results, and save that question as a model, which people can refer to in their queries just like they would with any other table.
```
Comment (maxzheng) [inline docs/questions/native-editor/referencing-saved-questions-in-queries.md:11]: I actually like users better, but ok if that's the new norm.

File: docs/questions/native-editor/referencing-saved-questions-in-queries.md [hunk 2]
```diff
@@ -8,48 +8,21 @@ redirect_from:
...
+Type `{% raw %}{{#your search term }} {% endraw %}` and Metabase will display a list of models and questions relevant to your search term.
 
-We'll select that question, and Metabase will update our code with the question's ID, `5`:
+![Typeahead search dropdown for referencing questions and models in SQL queries](../images/search-dropdown.png)
 
-```
-SELECT count(*)
-FROM {% raw %}{{#5-gizmo-orders-in-2019}}{% endraw %}
-```
+You can also find the question or model ID by navigating in Metabase to the model or question you'd like to reference in your query. The items' ID is in the URL in your browser's address bar. The ID will be the number after `/model/` or `/question/`. E.g., for `https://metabase.example.com/model/12345-example-name`, the model's ID would be `12345`.
```
Comment (maxzheng) [inline docs/questions/native-editor/referencing-saved-questions-in-queries.md:23]: item's? Or simply removing items and leave "The ID" would work too.

File: docs/questions/native-editor/referencing-saved-questions-in-queries.md [hunk 3]
```diff
@@ -8,48 +8,21 @@ redirect_from:
...
 
-This query returns the number of rows in our saved question.
+Only the `#` and `ID` is required. Metabase justo display the model or question name to make the query more readable.
```
Comment (maxzheng) [inline docs/questions/native-editor/referencing-saved-questions-in-queries.md:25]: typo on `justo` or replace with `only`.

**Review by maxzheng — APPROVED** [review]

File: docs/questions/native-editor/referencing-saved-questions-in-queries.md [hunk 4]
```diff
@@ -8,48 +8,21 @@ redirect_from:
...
-WHERE  p.category = 'Gizmo'
-       AND o.created_at BETWEEN '2019-01-01' AND '2019-12-31'
-```
-
-We'll save that question as "Gizmo orders in 2019", then we'll convert it to a [model][model].
-
-Now let's refer to "Gizmo orders in 2019" in a new query. To keep it simple, let's say we just want to count all of those Gizmo orders from 2019. We can use the `#` symbol to refer to a saved question in a query.
-
-If we type out:
+## Search for models and questions as you type
```
Comment (maxzheng) [inline docs/questions/native-editor/referencing-saved-questions-in-queries.md:13]: Is example-notebook.png in the old doc still used in other places? If not, then it's better to remove it too.

File: docs/questions/native-editor/referencing-saved-questions-in-queries.md [hunk 5]
```diff
@@ -8,48 +8,21 @@ redirect_from:
...
 
-For example, let's say we have a lot of data spread across a number of tables, but our users are most interested in a subset of that data. We can perform a complicated query once to return those results, and save that question as a model, which people can refer to in their queries just like they would with any other table.
+For example, let's say we have a lot of data spread across a number of tables, but our people are most interested in a subset of that data. We can perform a complicated query once to return those results, and save that question as a model, which people can refer to in their queries just like they would with any other table.
 
-Here's how it works. First, create and save a question that returns the result set you'd like to make available for people to query. Using the sample database included with Metabase as an example, let's say we want to provide a result set (a "table") that only has orders from 2019, and only includes orders for products in the Gizmo category.
-
-We could create this model using the notebook editor, like so:
-
-![Example notebook](../images/example-notebook.png).
```
Comment (maxzheng) [inline docs/questions/native-editor/referencing-saved-questions-in-queries.md:17]: Is example-notebook.png in the old doc still used in other places? If not, then it's better to remove it too.

**Comment by jeff-bruemmer** [comment]
@metabase-bot backport release-x.48.x

**Pull request merged**